lib/awful/screen.lua

```diff
@@ -534,75 +687,49 @@ function screen.set_auto_dpi_enabled(enabled)
     data.autodpi = enabled
 end
 
-
 --- The number of pixels per inch of the screen.
+--
+-- Unless manually overriden, the dpi will be computed using the output side is
+-- `autodpi` is `true` or otherwise the X11 fallback (usually 96).
```

Comment:
I fail to parse this sentence and I have no idea what "the output side" is supposed to refer to